diff --git a/Source/Release/makefile b/Source/Release/makefile
index 2241106..6248625 100644
--- a/Source/Release/makefile
+++ b/Source/Release/makefile
@@ -28,7 +28,7 @@ all: fftp
 fftp: $(OBJS) $(USER_OBJS)
 	@echo 'Building target: $@'
 	@echo 'Invoking: GCC C Linker'
-	gcc  -o "fftp" $(OBJS) $(USER_OBJS) $(LIBS)
+	$(CC) $(CFLAGS)  -o "fftp" $(OBJS) $(USER_OBJS) $(LIBS)
 	@echo 'Finished building target: $@'
 	@echo ' '
 
diff --git a/Source/Release/subdir.mk b/Source/Release/subdir.mk
index f520c4d..bcf6f32 100644
--- a/Source/Release/subdir.mk
+++ b/Source/Release/subdir.mk
@@ -26,7 +26,7 @@ C_DEPS += \
 %.o: ../%.c
 	@echo 'Building file: $<'
 	@echo 'Invoking: GCC C Compiler'
-	gcc -std=c99 -O3 -Wall -Wextra -c -fmessage-length=0 -Wno-unused-parameter -Wno-unused-result -fno-ident -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@)" -o "$@" "$<"
+	$(CC) $(CFLAGS) -std=c99 -O3 -Wall -Wextra -c -fmessage-length=0 -Wno-unused-parameter -Wno-unused-result -fno-ident -MMD -MP -MF"$(@:%.o=%.d)" -MT"$(@)" -o "$@" "$<"
 	@echo 'Finished building: $<'
 	@echo ' '
